ci/dangerfile.js

```diff
@@ -23,10 +23,10 @@ function validatePrTitle() {
     return
   }
 
-  fail(
+  warn(
```

Comment:
should we use `warn` instead of `fail`?

i was thinking that with `warn` it might be tempting to ignore the message, while with `fail` it will be mandatory to fix the title. WDYT?

Reply:
fail seems to require some extra permissions so that it can modify the PR status